pkgs/by-name/in/inshellisense/package.nix

```diff
@@ -21,7 +21,10 @@ buildNpmPackage rec {
   #
   # when `node-gyp` tries to build the dep. The below allows `npm` to download the prebuilt binary.
   makeCacheWritable = stdenv.isDarwin;
-  nativeBuildInputs = lib.optional stdenv.isDarwin cacert;
+  nativeBuildInputs = lib.optionals stdenv.isDarwin [
+    cacert
+    patchelf
```

Comment:
Why do we need patchelf on darwin?

Reply:
I added it because during the `fixupPhase` the build log contains dozens of lines saying:
```
/nix/store/h9lc1dpi14z7is86ffhl3ld569138595-audit-tmpdir.sh: line 23: patchelf: command not found
```

The build does succeed without it, and I'm not sure why it's needed, but it made those messages go away which seemed good? ¯\_(ツ)_/¯

Reply:
So the problem is that there are elf binaries in the package output despite targeting darwin? We can safely ignore these errors.

Reply:
Ok, I removed it.